app/models/course.rb

```diff
@@ -0,0 +1,15 @@
+class Course < ApplicationRecord
+  belongs_to :creator, class_name: "User", foreign_key: "created_by"
+
+  has_many :lessons, dependent: :destroy
+
+  has_many :user_courses, dependent: :destroy
+
+  has_many :users, through: :user_courses
+
+  has_many :admin_course_managers, dependent: :destroy
+
+  has_many :admins, through: :admin_course_managers, source: :user
```

Comment:
```suggestion
  has_many :lessons, dependent: :destroy
  has_many :user_courses, dependent: :destroy
  has_many :users, through: :user_courses
  has_many :admin_course_managers, dependent: :destroy
  has_many :admins, through: :admin_course_managers, source: :user
```
Các has_many này cùng thành phần thì k cần line trống, e fix tương tự những chỗ khác nha